.gitignore

```diff
@@ -7,3 +7,4 @@ lib
 dist
 .env
 coverage
+packages/**/lincense.md
```

Comment:
do we need a license file in each package?

Reply:
It just copies it when the build script runs from the root path. I think every npm package should have a LICENSE file, I don't think we lose anything without it.